From c1028d9b0fd1953f9aefc4377e4afdba59833000 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Tue, 5 Sep 2017 18:07:38 +0800
Subject: [PATCH 3310/5242] MLK-17491-7 pinctrl: imx: remove unnecessary
 fsl,mux_mask property

commit  a8b54542fe26cd743d5684e8aced5b96c6110577 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove unnecessary fsl,mux_mask property which is also not documented in
binding doc. As we already have imx_pinctrl_soc_info structure which
represents the SoC specific properties, encode in it instead.

The patch also simplies the code a bit by removing the mux_shift
calculation code which is not necessary as well.

Reviewed-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 1d78563..121a1d4 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -514,14 +514,12 @@
 		iomuxc: iomuxc@4103D000 {
 			compatible = "fsl,imx7ulp-iomuxc-0";
 			reg = <0x4103D000 0x1000>;
-			fsl,mux_mask = <0xf00>;
 			status = "disabled";
 		};
 
 		iomuxc1: iomuxc1@40ac0000 {
 			compatible = "fsl,imx7ulp-iomuxc-1";
 			reg = <0x40ac0000 0x1000>;
-			fsl,mux_mask = <0xf00>;
 		};
 
 		gpio0: gpio@40ae0000 {
-- 
1.7.9.5

